src/pages/map.tsx

```diff
@@ -0,0 +1,72 @@
+import {
+  Tab,
+  Tabs,
+  TabList,
+  TabPanel,
+  TabPanels,
+  Heading,
+  Stack,
+  Box,
+  Text,
+  Spinner,
+} from '@chakra-ui/react';
+import React, { useState } from 'react';
+import { trpcNext } from "../trpc";
+import { Landmark } from 'shared/Map';
+
+export default function Page() {
+  const [tabIndex, setTabIndex] = useState(0);
+
+  const personalGrowth = trpcNext.map.list.useQuery('个人成长', { enabled: tabIndex === 0 });
+  const careerDevelopment = trpcNext.map.list.useQuery('事业发展', { enabled: tabIndex === 1 });
+  const socialResponsibility = trpcNext.map.list.useQuery('社会责任', { enabled: tabIndex === 2 });
```

Comment:
Do not fetch everything here. Use lazy loading Tabs and encapsulate data within individual tabs. Also use a common tab component to reuse code. Don't repeat yourself.